packages/components/socialFeed/NewsFeed/MapComponent/Map.native.tsx

```diff
@@ -0,0 +1,31 @@
+import { LatLng, LeafletView } from "react-native-leaflet-view";
+type Tlocation = [number, number];
+
+const MapMobile = ({ locationSelected }: { locationSelected: Tlocation }) => {
+  const DEFAULT_COORDINATE: LatLng = {
+    lat: locationSelected[0],
+    lng: locationSelected[1],
+  };
+
+  return (
+    <LeafletView
+      renderLoading={() => <></>}
+      zoom={12}
+      mapCenterPosition={DEFAULT_COORDINATE}
+      mapLayers={[
+        {
+          url: "https://{s}.tile.jawg.io/jawg-dark/{z}/{x}/{y}{r}.png?access-token=QkwJFLzzxPan25YCgnDExGpMFPxA3x4lnyKiUf8zmaqXLP5XyOR8n3yEM8jlKV3W",
```

Comment:
this token should be:

- rotated. As this one has been published in github
- made into a ENV VAR